RPIPs/RPIP-42.md

```diff
@@ -22,19 +22,35 @@ This proposal dramatically increases the LTV used in the protocol (loan to value
 This work is based on prior work; a copy can be found [here](../assets/rpip-42/bond_curves.md).
 
 ## Specification
-- The protocol SHALL be able to penalize stake at the node level
-- Node Operators with <`base_mp_num` base minipools SHALL be able to make a base minpool
-  - A base minipool SHALL have a `base_mp_bond` bond from the Node Operator
-- Node Operators with >=`base_mp_num` base minipools SHALL be able to make a satellite minipool
-  - A satellite minipool SHALL have a `satellite_mp_bond` bond from the Node Operator
-- If a Node has satellite minipools active while that Node has <`base_mp_num` base minipools active, funds SHALL NOT be withdrawable by the Node Operator
-  - It is NOT RECOMMENDED to exit base minipools unless all satellite minipools have been exited
-  - A user MAY exit satellite minipools to allow for fund withdrawal 
-  - The protocol MAY or MAY NOT provide a path to upgrade a satellite minipool to a base minipool (to allow for fund withdrawal)
+- The oDAO SHALL be able to penalize stake at the node level when a [Penalizable offense](#penalizable-offenses) is committed
+- Legacy minipool deposits SHOULD be disabled
+- Node Operators SHALL NOT be required to stake RPL in order to create validators within their megapool
+- When Node Operators create validators:
+  - If fewer than `base_num` validators already exist, the required `user_deposit` is `base_bond` per validator.
+  - If at least `base_num` validators already exist, the required `user_deposit` is `reduced_bond` per validator.
```

Comment:
Was this meant to be the same? Should there not be a distinction between < base = 4 ETH and >= base at 1.5 ETH. (Or the ETH number that we specify). 
Also the way this is written base = 3 in the 4+4+1.5 model.

Reply:
This was Val's clarification:
Seems like it works just fine right now? Let's use base_num=2, base_bond=4, reduced_bond=1.5 and https://github.com/Valdorff/RPIPs/blob/spring2024-tokenomics-rework-drafts2/RPIPs/RPIP-42.md#specification.

Now let's make some deposits:
You have 0 validators, which is <2, so your user_deposit is 4
You have 1 validator, which is <2, so your user_deposit is 4
You have 2 validators, which is >=2, so your user_deposit is 1.5
You have n validators, which is >=2, so your user_deposit is 1.5